spec/models/articles_db_view_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ArticlesDbView, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
Pending test

Reply:
Test has been complete in the further commit